pkg/obfuscate/redis.go

```diff
@@ -245,6 +245,43 @@ func obfuscateRedisCmd(out *strings.Builder, cmd string, args ...string) {
 	out.WriteString(strings.Join(args, " "))
 }
 
+// removeAllRedisArgs will take in a command and obfuscate all arguments following
+// the command, regardless of if the command is valid Redis or not
+func (*Obfuscator) RemoveAllRedisArgs(rediscmd string) string {
+	fullCmd := strings.Fields(rediscmd)
+	if len(fullCmd) == 0 {
+		return ""
+	}
+	cmd, args := fullCmd[0], fullCmd[1:]
+
+	var out strings.Builder
+	out.WriteString(cmd)
+	if len(args) == 0 {
+		return out.String()
+	}
+
+	out.WriteByte(' ')
+	switch strings.ToUpper(cmd) {
+	case "BITFIELD":
+		out.WriteString("?")
+		for _, arg := range args {
+			if arg == "SET" || arg == "GET" || arg == "INCRBY" {
```

Comment:
Above, we checked `strings.ToUpper(cmd)`.
Could these command args also be lower-case?